Airports/server.js

```diff
@@ -25,18 +24,22 @@ proxyRouter.get('/\*', (req, res) => {
     }
     const url = 'https://osdatahubapi.os.uk' + req.url + separator + 'key=' + key;
```

Comment:
I see this hasn't been edited, but it looks like this would concatenate 2 urls?

Reply:
Nope... the url that Express gives to us is just the path bit... perhaps we should rename the variable